siap/siap_client/utils.py

```diff
@@ -292,21 +291,24 @@ def _type_operation(type_operation_from_siap: str) -> TypeOperation:
 
 
 def _nature_logement(nature_logement_from_siap: str) -> TypeOperation:
-    nature_logement = NatureLogement.LOGEMENTSORDINAIRES
     if nature_logement_from_siap in ["ALF", NatureLogement.AUTRE]:
         nature_logement = NatureLogement.AUTRE
-    if nature_logement_from_siap in ["HEB", NatureLogement.HEBERGEMENT]:
+    elif nature_logement_from_siap in ["HEB", NatureLogement.HEBERGEMENT]:
         nature_logement = NatureLogement.HEBERGEMENT
-    if nature_logement_from_siap in ["RES", NatureLogement.RESISDENCESOCIALE]:
+    elif nature_logement_from_siap in ["RES", NatureLogement.RESISDENCESOCIALE]:
         nature_logement = NatureLogement.RESISDENCESOCIALE
-    if nature_logement_from_siap in ["PEF", NatureLogement.PENSIONSDEFAMILLE]:
+    elif nature_logement_from_siap in ["PEF", NatureLogement.PENSIONSDEFAMILLE]:
         nature_logement = NatureLogement.PENSIONSDEFAMILLE
-    if nature_logement_from_siap in ["REA", NatureLogement.RESIDENCEDACCUEIL]:
+    elif nature_logement_from_siap in ["REA", NatureLogement.RESIDENCEDACCUEIL]:
         nature_logement = NatureLogement.RESIDENCEDACCUEIL
-    if nature_logement_from_siap in ["REU", NatureLogement.RESIDENCEUNIVERSITAIRE]:
+    elif nature_logement_from_siap in ["REU", NatureLogement.RESIDENCEUNIVERSITAIRE]:
         nature_logement = NatureLogement.RESIDENCEUNIVERSITAIRE
-    if nature_logement_from_siap in ["RHVS", NatureLogement.RHVS]:
+    elif nature_logement_from_siap in ["RHVS", NatureLogement.RHVS]:
         nature_logement = NatureLogement.RHVS
+    elif nature_logement_from_siap in ["LOO", NatureLogement.LOGEMENTSORDINAIRES]:
+        nature_logement = NatureLogement.LOGEMENTSORDINAIRES
+    else:
+        raise InconsistentDataSIAPException("Missing NatureLogement")
```

Comment:
Ça pourrait être utile de passer à l'exception la valeur passée en paramètre pour débuguer l'exception 
```suggestion
        raise InconsistentDataSIAPException(f"The NatureLogement value coming from SIAP is missing or unexpected : {nature_logement_from_siap}")
```

Reply:
Après réflexion, c'est possible que Sentry soit en mesure de nous aider à debuguer l'exception.  
Je ne l'utilise pas systématiquement et j'aime bien avoir les valeurs d'exception dans les logs directement, mais avec Sentry je crois qu'on a accès au contexte de survenue de l'exception 🤔